src/Accelerators/NNPA/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(DLC_SRC_ROOT "${CMAKE_CURRENT_SOURCE_DIR}")
+set(DLC_BIN_ROOT "${CMAKE_CURRENT_BINARY_DIR}")
+
+set(DLC_LIBRARY_PATH ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+set(DLC_RUNTIME_PATH ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+
+set(DLC_ONNX_MLIR_SRC_ROOT ${ONNX_MLIR_SRC_ROOT})
+set(DLC_ONNX_MLIR_BIN_ROOT ${ONNX_MLIR_BIN_ROOT})
+
+# The path to find the zDNN library.
+if (DEFINED ENV{ZDNN_LIBRARY_DIR} AND EXISTS $ENV{ZDNN_LIBRARY_DIR})
```

Comment:
How is the library shipped? Even if it doesn't have cmake files, there might be ways for cmake to find it.

Reply:
This PR adds it to third_party. It seems to vave a regular makefile, but I haven't looked at it.

Reply:
I see, so the source is added to `third_party`, I can't seem to figure out how it is built and these environment paths are set?